README.md

```diff
@@ -173,9 +184,44 @@ terraform {
+- Continuous Improvement: Update documentation and constraints (versions.tf) as Terraform and providers evolve, and as you refine the module’s functionality. -->
+
+## TF Linting + CI
+
+There are many tools to format, lint, and ensure consistency of TF code. The tool that we recommend is [trunk Code Quality](https://docs.trunk.io/code-quality). This single tool allows us to do the following:
+
+1. Format our TF code with `terraform fmt` or `tofu fmt` within our IDE and ensure this is run on each commit.
+   1. [This is handled by the trunk `terraform` or `tofu` linter](https://docs.trunk.io/code-quality/linters/supported/tofu).
+2. Validate our TF code with `terraform validate` or `tofu validate` within our IDE and ensure this is run on each commit.
+   1. [This is handled by the trunk `terraform` or `tofu` linter](https://docs.trunk.io/code-quality/linters/supported/tofu).
+3. Generate documentation for our TF code with `terraform-docs` and ensure it is kept up-to-date on each commit.
+   1. [This is handled by the trunk `terraform-docs` action](https://github.com/trunk-io/plugins/tree/main/actions/terraform-docs), which [Masterpoint originally developed](https://github.com/trunk-io/plugins/pull/966).
+4. Run TFLint against our code to ensure it is written against the best practices.
+   1. [This is handled by the trunk `tflint` linter](https://docs.trunk.io/code-quality/linters/supported/tflint).
+5. Run a TF security scan against our code to ensure we're not introducing any security vulnerabilities.
+   1. [This is handled by the trunk `trivy` linter](https://docs.trunk.io/code-quality/linters/supported/trivy).
+6. Run these checks in a CI pipeline to ensure they're enforced on each PR.
+   1. [This is handled by the trunk-action workflow](https://docs.trunk.io/code-quality/setup-and-installation/github-integration) in the [.github/workflows/lint.yaml](.github/workflows/lint.yaml) file.
+
+As you can see, this is a LOT of checks that trunk is supporting for us and this consolidation on one tool to support this (and much more) is a huge win.
+
+Check out our [.trunk/trunk.yaml](.trunk/trunk.yaml) file to see how we configure this and [check the trunk Code Quality getting started documentation](https://docs.trunk.io/code-quality) on how you can use this tool for your own project.
+
+## Frequently Asked Questions
+
+### Why do you prefer DRY (Don't Repeat Yourself) Root Modules vs WET (Write Every Time) Root Modules?
+
+We believe IaC should be treated like code and traditional software. You encode some level of business logic into it and consistently copying+pasting that logic as part of your day-to-day processes creates a maintenance burden that we believe should be avoided. This particularly shows up when IaC is managed at scale and we have seen these types of setups lead to a lot of toil and a lot of technical debt.
```

Comment:
```suggestion
We advocate IaC should be treated like code and traditional software. You encode some level of business logic into it, and consistently copying+pasting that logic as part of your day-to-day processes creates a maintenance burden that we believe should be avoided. This particularly shows up when IaC is managed at scale and we have seen these types of setups lead to a lot of toil and a lot of technical debt.
```
Replace to avoid duplications.